index.go

```diff
@@ -405,9 +405,8 @@ func (p indexSlice) Less(i, j int) bool { return p[i].Name() < p[j].Name() }
 
 // IndexInfo represents schema information for an index.
 type IndexInfo struct {
-	Name    string       `json:"name"`
-	options IndexOptions `json:"options"`
```

Comment:
this is kind of interesting... perhaps this was unexported by mistake? seems like we might want to return options with index info

Reply:
It's actually not referred to by anything, hence the removal.

Reply:
right, but it would be useful in an HTTP response serialized to json. I think this is the struct that gets returned if you make /index/name GET request.

Reply:
I made a separate PR to add that feature.